cors_test.go

```diff
@@ -0,0 +1,166 @@
+package gin
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	configmocks "github.com/goravel/framework/contracts/config/mocks"
+	contractshttp "github.com/goravel/framework/contracts/http"
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
Please sort import.

Reply:
`github.com/goravel/framework` and `github.com/stretchr/testify` are both third packages, so they can be together. The import order is: 
```
system packages

third packages

self packages
```